src/Connection.js

```diff
@@ -0,0 +1,52 @@
+
+define(["ObjectsConfig"], function (ObjectsConfig) {
+    "use strict";
```

Comment:
move use strict at the top of the script
=> to set the script mode to the whole script